### PR TITLE
[SPARKTA-496] [PLUGINS] [WEB] Bug with default Kafka port in output

### PR DESCRIPTION
**Pull request by compae**
Added new kafka properties for support all producer options

***given*** an instance of Sparta with one output kafka
***when*** you add this output into triggers or outputs
***then*** return the aggregated data or trigger data must be sended to kafka

***given*** an instance of kafkaOutput
***when*** you want to add more kafka properties
***then*** you must have this option on the sparta frontend

### CODE REVIEW

File: plugins/src/main/scala/com/stratio/sparta/plugin/output/kafka/producer/KafkaProducer.scala [hunk 1]
```diff
@@ -68,6 +71,23 @@ object KafkaProducer {
     }
   }
 
+  private def getAdditionalOptions(properties: Map[String, JSerializable]): Map[String, JSerializable] = {
```
Comment (acemilio) [inline plugins/src/main/scala/com/stratio/sparta/plugin/output/kafka/producer/KafkaProducer.scala]: I think you could try to refactor getAdditionalOptions and getList functions

File: plugins/src/main/scala/com/stratio/sparta/plugin/output/kafka/producer/KafkaProducer.scala [hunk 2]
```diff
@@ -68,6 +71,23 @@ object KafkaProducer {
...
+    Try(properties.getMapFromJsoneyString(DefaultPropertiesKey)) match {
+      case Success(jsonObject) =>
+        val valueAsSeq = jsonObject.map(c =>
+          (c.get(PropertiesKey) match {
+            case Some(value) => value.toString
```
Comment (anistal) [inline plugins/src/main/scala/com/stratio/sparta/plugin/output/kafka/producer/KafkaProducer.scala]: Could you change it with getOrElse?
Is it necessary a toString in the value?

**Comment by anistal** [comment]
Add a definition of done too. Thanks.

**Comment by anistal** [comment]
+1

**Comment by acemilio** [comment]
+1

**Comment by antonioalf** [comment]
:shipit:

**Comment by stratiocommit** [comment]
Comment accepted. Will treat it ASAP

**Comment by stratiocommit** [comment]
Squashing and checks in progress.

**Pull request merged**